src/main/scala/com/download/Logging.scala

```diff
@@ -0,0 +1,7 @@
+package com.download
+
+import akka.event.jul.Logger
+
```

Comment:
This things saves you a lot of repeating
val logger = Logger(this.getClass.getSimpleName)
all over the place.